storage/gcp/gcp_test.go

```diff
@@ -368,10 +368,7 @@ func TestStreamEntries(t *testing.T) {
 	// Populate entry bundles:
 	// first to logSize1 (so we're sure we've got the partial bundle)
 	for r, idx := logSize1, uint64(0); r > 0; idx++ {
-		sz := layout.EntryBundleWidth
-		if sz > r {
-			sz = r
-		}
+		sz := min(layout.EntryBundleWidth, r)
```

Comment:
I find swapping the ordering around (`thing_which_is_growing`, `constant_limit`) makes it a little easier to read for some reason:

```suggestion
		sz := min(r, layout.EntryBundleWidth)
```